### ISSUE TITLE
todo: Port general theme information here

### ISSUE THREAD

**Issue by bigpick** [spaceduck-theme/spaceduck#1]
# Info
Port general theme information here, from https://github.com/pineapplegiant/spaceduck

... not including the vim/plugin stuff, since that will be in a `spaceduck-theme/vim` dedicated repo (but am not going to try moving that until coordinating with overlord and jinho, since want to wait for the latest dev branch to be merged into pineapplegiant's master before importing)

**Comment by pineapplegiant** [comment]
Thanks for your work mate. Yeah we'll have to do some work there, but this could be the central hub, along with holding the palette information.

**Comment by bigpick** [comment]
+1. I think would be a good place to hold the specification too (which imo would be a great addition, to help steer consistency in future/current ports)

**Comment by spcbfr** [comment]
@pineapplegiant I think you should include in a wiki or somewhere else information on what each color reperesents, not only techniquel stuff (example blue/green means go, orange is for warning the user, the main purple is for catching the user's attention etc...)

**Comment by bigpick** [comment]
Yeah, a good specification would cover technical things (like syntax groups and specific identifiers and things so that you can fill out things like atom themes and vs code themes and other language based things) as well as general usage ("info"/"danger"/"error" type things and button states, etc)
